File: ios/xiuyixiu/xiuyixiu/ViewController.m
//
//  ViewController.m
//  xiuyixiu
//
//  Created by parkinwu on 16/9/5.
//  Copyright © 2016年 parkinwu. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIWebView *webView;

@end

@implementation ViewController
#define   DEGREES_TO_RADIANS(degrees)  ((M_PI * degrees)/ 180)
- (void)viewDidLoad {
    [super viewDidLoad];
    
    NSString * path = [[NSBundle mainBundle] pathForResource:@"index" ofType:@"html" inDirectory:@"www"];
    NSURL * url = [NSURL URLWithString:path];
    [self.webView loadRequest:[NSURLRequest requestWithURL:url]];
    
    
    
}



- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: ios/router/router/ViewController.swift
//
//  ViewController.swift
//  router
//
//  Created by parkinwu on 2016/9/27.
//  Copyright © 2016年 parkinwu. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    @IBAction func toFirst(sender: AnyObject) {
        let url = URL(string: "MyApp://?token=123")
        
        if UIApplication.shared.canOpenURL(url!) {
            UIApplication.shared.openURL(url!)
        }
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

